apps/src/weblab/WebLab.js

```diff
@@ -98,6 +98,7 @@ WebLab.prototype.init = function (config) {
   config.centerEmbedded = false;
   config.wireframeShare = true;
   config.noHowItWorks = true;
+  config.baseShareUrl = 'https://codeprojects.org';
```

Comment:
This isn't going to work in any environment other than production, right?  Do we have any sort of test environment parallel to codeprojects.org?

If we do have one, maybe we should make this a `CDO` environment-configurable variable (in deployment.rb).